actionpack/test/controller/render_xml_test.rb

```diff
@@ -92,11 +92,11 @@ def test_should_render_formatted_xml_erb_template
 
   def test_should_render_xml_but_keep_custom_content_type
```

Comment:
@y-yagi : Should we rename the test description here from `content_type` to `media_type`? I made the changes but was not sure to raise PR. 

At other places, it looked fine to me. Only test desc here should maybe change.

Reply:
This is test is to confirm that `content_type` takes precedence when specifying `render` and `content_type`. 
https://github.com/rails/rails/blob/90d6237762da832d0c6302b31fd68d3efb771c64/actionpack/test/controller/render_xml_test.rb#L39
So I think the current test name is appropriate.